src/services/utilities.ts

```diff
@@ -2157,6 +2157,15 @@ namespace ts {
         return { text, kind: SymbolDisplayPartKind[kind] };
     }
 
+    /**
+     * Functions that call displayPart with specific `SymbolDisplayPartKind`
+     */
+    export type ConstructSpecificDisplayPartFunction = (text: string) => ReturnType<typeof displayPart>;

```

Comment:
```suggestion
    export type ConstructSpecificDisplayPartFunction = (text: string) => SymbolDisplayPart;
```

also I would inline this; the name is longer than the type.